lib/trento_web/controllers/cluster_controller.ex

```diff
@@ -3,19 +3,20 @@ defmodule TrentoWeb.ClusterController do
 
   alias Trento.{Clusters, Hosts}
 
-  alias Trento.Integration.Checks
+  alias Trento.Integration.Checks, as: ChecksIntegration
```

Comment:
I would personally leave this as `Checks` and set the other as `ChecksOpenApi`, but well, not a big difference hehe

Reply:
I see, and I also tried to do that, but failed miserably :sweat_smile: 

```elixir
alias Trento.Integration.Checks, as: ChecksIntegration

alias TrentoWeb.OpenApi.Schema.{Checks, Cluster, Common}
```

I found it easier to alias that lonely `Trento.Integration.Checks` rather than `alias TrentoWeb.OpenApi.Schema.{Checks, Cluster, Common}` and if I understood correctly it is not possible to alias module names in multiple alias.

An alternative would be to split this
```elixir
alias TrentoWeb.OpenApi.Schema.{Checks, Cluster, Common}
```
to this
```elixir
alias TrentoWeb.OpenApi.Schema.Checks, as: ChecksSchema
alias TrentoWeb.OpenApi.Schema.Cluster, as: ClusterSchema
alias TrentoWeb.OpenApi.Schema.Common, as: CommonSchema
```

I just went for what was more immediate :sweat_smile:

Reply:
@fabriziosestito can probably enlight us

Reply:
@nelsonkopliku 

`alias TrentoWeb.OpenApi.Schema`

and then `Schema.Cluster`